scripts/tests/test_pjdfstest.sh

```
@@ -0,0 +1,203 @@
+  fi
+  echo "Building pjdfstest (release) ..."
+  (cd "$PJDFSTEST_DIR/rust" && cargo build --release --quiet)
+  PJDFSTEST_BIN="$PJDFSTEST_DIR/rust/target/release/pjdfstest"
+  test -x "$PJDFSTEST_BIN" || { echo "FAIL: pjdfstest binary not built"; exit 1; }
+}
+
+create_seed_archive() {
+  # pna create requires at least one input. Use a relative path so the entry
+  # name is just `seed.txt` (not absolute), matching the convention in the
+  # other test scripts.
+  (cd "$WORKDIR" && echo "seed" > seed.txt && \
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)